set.mm

```diff
@@ -427289,8 +427290,8 @@ All axiomatic assertions ($a statements)
        a wff or class.  Metamath blindly treats $a statements as new
        given facts but does not try to justify them.  The mmj2 program
        will justify the definitions as sound as discussed below,
-       except for 4 definitions ( ~ df-bi , ~ df-cleq , ~ df-clel , ~ df-clab )
-       that require a more complex metalogical justification by hand.
+       except for four of them ( ~ df-bi , ~ df-clab , ~ df-cleq , ~ df-clel )
+       that require a more complex metalogical justification.
```

Comment:
Why did you remove "by hand"? How atre they justified otherwise?

Reply:
I think "by hand" adds nothing to the sentence. (Also, that "more complex metalogical justification" could be made by a human, or by a more complex algorithm in the future, or be formalized in a "metamath-in-metamath" database or in Mario's metamath0... So, removing "by hand" is more about being agnostic about how that justification may be done.)

Reply:
But how are the justified currently, i.e., within or at least for set. mm? If there were not justified yet, most of set.mm would be meaningless.

Reply:
I suppose to be more specific, we can add a link explaining the current metalogical justification in more detail

Reply:
The justification is in Lévy's Basic Set Theory, Appendix, as written in the homepage mmset.html and discussed on the google discussion group multiple times.

Anyway, I'll revert my change since I don't want to spend more time on two words.